src/browser/tab/mod.rs

```diff
@@ -380,4 +377,34 @@ impl<'a> Tab {
         })?;
         Ok(self)
     }
+
+    /// Tells Chrome to start tracking which lines of JS have been executed.
```

Comment:
I'd suggest a unified wording of `Start tracking which lines of JS have been executed` instead of `Tells Chrome...`.